src/components/watchers-form.vue

```diff
@@ -36,6 +36,7 @@
               :helper="$t('Select the variable to watch on this screen or type any request variable name')"
               @open="loadVariables"
               @tag="addTag"
+              :tag-placeholder="$t('Press enter to use this variable')"
```

Comment:
I added this line to change the message from "Press enter to create a tag" to "Press enter to use this variable".

Before:
<img width="815" alt="Screen Shot 2021-05-12 at 10 04 04 AM" src="https://user-images.githubusercontent.com/867714/118015988-b8b6d700-b309-11eb-9548-949b4fbff0b0.png">

After:
<img width="815" alt="Screen Shot 2021-05-12 at 10 04 43 AM" src="https://user-images.githubusercontent.com/867714/118016008-beacb800-b309-11eb-994a-e8a87c47ecdd.png">